src/main.js

```diff
@@ -1,10 +1,6 @@
 export default function ctl(template) {
-  var trimmedClassnames = template.replace(/\s+/gm, " ");
-  var formattedClassnames = trimmedClassnames
-    .split(" ")
-    .filter((c) => c !== "false" && c !== "true" && c !== "undefined")
-    .join(" ")
-    .trim();
-
-  return formattedClassnames;
-}
+  return template
+    .replace(/\s(undefined|false|true)\s/g, '')
+    .replace(/\s+/g, ' ')
+    .trim()
+}
```

Comment:
🔍 This series of `.replace()` calls will
1. Remove all occurrences of `undefined`, `false`, or `true` from the string
2. Replace any space-type characters with a single whitespace